src/modules/ContactPage/components/molecules/ContactUsForm/index.tsx

```diff
@@ -0,0 +1,158 @@
+              <Controller
+                name='subject'
+                control={control}
+                render={({ field }) => {
+                  return (
+                    <SelectInput
+                      {...field}
+                      errorMessage={errors['subject']?.message}
+                      error={Boolean(errors['subject'])}
+                      aria-label='select_subject'
+                      label='Subject'
+                      options={[
+                        {
+                          label: 'Subject One',
+                          value: 'first',
+                        },
+                        {
+                          label: 'Subject Two',
+                          value: 'second',
+                        },
+                      ]}
+                      placeholder='Choose'
+                    />
+                  );
+                }}
+              />
+            </div>
+            <div className='w-full medium:w-1/2'>
+              <Controller
+                name='name'
+                control={control}
+                render={({ field }) => {
+                  return (
+                    <InputField
+                      errorMessage={errors['name']?.message}
+                      error={Boolean(errors['name'])}
+                      className='my_input'
+                      id='name'
+                      label='name'
+                      placeholder='Enter your name'
+                      {...field}
+                    />
+                  );
+                }}
+              />
+            </div>
+          </div>
+          <div>
+            <Controller
+              name='email'
+              control={control}
+              rules={{ required: true }}
+              render={({ field }) => {
+                return (
+                  <InputField
+                    errorMessage={errors['email']?.message}
+                    error={Boolean(errors['email'])}
+                    id='email'
+                    label='email'
+                    placeholder='Enter your email address'
+                    {...field}
+                  />
+                );
+              }}
+            />
+          </div>
+          <div>
+            <Controller
+              name='message'
+              control={control}
+              render={({ field }) => {
+                return (
+                  <InputField
+                    id='textarea'
+                    errorMessage={errors['message']?.message}
+                    error={Boolean(errors['message'])}
+                    label='message'
+                    multiline
+                    placeholder='Start typing'
+                    {...field}
+                  />
+                );
+              }}
+            />
+          </div>
+          <div className='inline-flex justify-end'>
+            <Button type='submit' className='inline-block' size='lg'>
```

Comment:
Submit button is taking full width on mobile screen 
```suggestion
            <Button type='submit' className='inline-block w-full medium:w-auto' size='lg'>
```